src/components/Films/index.tsx

```diff
@@ -1,37 +1,56 @@
-import { FC, useState } from 'react'
-type Props = { films: string[], updateList: (item: string[]) => void }
+import { FC, useState } from 'react';
+import styles from './FilmsList.module.scss';
+import clsx from 'clsx'; // assign a class with condition `clsx(false && styles.li)`
```

Comment:
vice-versa

Reply:
do you mean it's more accurate to say "assign a class depending on certain condition"?

Reply:
I meant that you need to swap lines 2 and 3